config.py

```diff
@@ -36,6 +36,15 @@ class Config(object):
     RAS_FRONTSTAGE_API_SERVICE = '{}://{}:{}/'.format(RAS_FRONTSTAGE_API_PROTOCOL,
                                                       RAS_FRONTSTAGE_API_HOST,
                                                       RAS_FRONTSTAGE_API_PORT)
+
+    RAS_SECURE_MESSAGING_PROTOCOL = os.getenv('RAS_SECURE_MESSAGING_PROTOCOL', 'http')
+    RAS_SECURE_MESSAGING_HOST = os.getenv('RAS_SECURE_MESSAGING_HOST', 'localhost')
+    RAS_SECURE_MESSAGING_PORT = os.getenv('RAS_SECURE_MESSAGING_PORT', 5050)
+
+    RAS_SECURE_MESSAGING_SERVICE = '{}://{}:{}/'.format(RAS_SECURE_MESSAGING_PROTOCOL,
```

Comment:
These shoud be RAS_SECURE_MESSAGE_SERVICE_PORT ect, to coincide what's in Jenkins